PhysicsTools/PatUtils/interface/PATJetCorrExtractor.h

```diff
@@ -88,6 +69,24 @@ class PATJetCorrExtractor {
 
     return corrJetP4;
   }
+
+private:
+  inline std::string format_vstring(const std::vector<std::string>& v) const {
```

Comment:
```static``` instead of ```const```?

Reply:
ok thanks, I will update to use `static`